programming/python/index.md

```diff
@@ -20,11 +20,10 @@ There are a number of tutorials out there which might help you to learn to progr
     there is a lot there and it may be a little overwhelming.
 
 * A number of tutorials for beginners are linked to from [here](http://wiki.python.org/moin/BeginnersGuide/NonProgrammers).
-    [Byte of Python](http://www.swaroopch.com/notes/Python_en:Functions) seems like a good one.
+    [Wikibooks Tutorial](https://en.wikibooks.org/wiki/Non-Programmer%27s_Tutorial_for_Python_2.6) seems like a good one.
```

Comment:
This tutorial claims to be for Python 2.6; that's really rather old at this point (and IIRC missing many of the features of Python 2.7). Are there other tutorials we could link to instead? (or maybe put this one a little lower down?)